docs/templates/10-introduction.ftl

```diff
@@ -0,0 +1,6 @@
+# Introduction
+
+${polaris_product_name} helps security and development teams analyze security risks
+in their software products. The ${solution_name} plugin enables you to invoke ${polaris_product_name}
```

Comment:
Maybe we should add a separate term to distinguish running the "Synopsys Polaris CLI" (the CLI) and uploading to "Synopsys Polaris" (the server)